src/main/docker/cypress/Coverage-Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM artemis:coverage-latest
+
+RUN echo "Installing needed dependencies" \
+  && apt-get update && apt-get install -y --no-install-recommends unzip \
```

Comment:
Maybe this is a misunderstanding on my side (since I saw this missing in multiple docker files in context of Artemis), but from my understanding `update` only checks if new versions are available without actually installing them. To do the actual update you need to also call `upgrade`.
```suggestion
  && apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends unzip \
```

Reply:
This is lifted from other artemis dockerfiles so that might be the reason.
https://github.com/ls1intum/Artemis/blob/b6f6ae519f6f039bd5791a7efccd0382827e70c2/src/main/docker/Dockerfile#L10-L13
Update indeed loads the latest package information, mainly to make sure the package about to be installed is the latest one in this case.
Since I don't actually want to change anything but get this utility, I think it is fine to only update @Strohgelaender